pootle/apps/pootle_project/migrations/0007_migrate_localfiletype.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+from pootle.core.delegate import formats
+
+
+def migrate_localfiletype(apps, schema_editor):
+    format_registry = formats.get()
+
+    projects = apps.get_model("pootle_project.Project").objects.all()
+    filetypes = apps.get_model("pootle_format.Format").objects.all()
+
+    for project in projects:
+        filetype = filetypes.get(
+            pk=format_registry[project.localfiletype]['pk'])
```

Comment:
this was discussed - basically we only allow `po` and `xliff` atm so its a safe assumption that it will be